hw/ip/edn/doc/theory_of_operation.md

```diff
@@ -27,9 +27,10 @@ In this mode, the EDN automatically issues a special reduced-latency `instantiat
 This means, for instance, that no personalization strings or additional data may be passed to the CSRNG application interface port in this mode.
 On exiting, the EDN issues an `uninstantiate` command to destroy the associated CSRNG instance.
 
-Once firmware initialization is complete, it is important to exit this mode if the endpoints ever need FIPS-approved random values.
+Once firmware initialization is complete, it is important to exit this mode if the endpoints ever need FIPS-approved random values. Should another generate command be needed, it can only be issued after exiting boot mode.
 This is done by either *clearing* the `EDN_ENABLE` field or *clearing* the `BOOT_REQ_MODE` field in [`CTRL`](registers.md#ctrl) to halt the boot-time request state machine.
-Firmware must then wait for successful the shutdown of the state machine by polling the `REQ_MODE_SM_STS` field of the [`SUM_STS`](registers.md#sum_sts) register.
+Firmware must then wait for the transition of the state machine by polling the `CMD_RDY` field of the [`SW_CMD_STS`](registers.md#sw_cmd_sts) register or wait
+for the state machine to enter the SW mode by polling the [`MAIN_SM_STATE`](registers.md#main_sm_state) register.
```

Comment:
I think we need to discuss this as soon as your back. From the description of the PR, related issue and existing documentation, I understood that boot mode should be exited automatically once the boot sequence ends. What the implementation and this part of the documentation suggest is that the boot mode is exited only when software explicitly requests that. What do we want? Let's discuss next week.